### PR TITLE
Add appveyor builds

### PR DESCRIPTION
**Pull request by mercuriete**
We can build dll 32 and 64 bits using appveyor.

example of url:

jobs: https://ci.appveyor.com/project/mercuriete/cchromaeditor/builds/35941481

64 bits https://ci.appveyor.com/api/buildjobs/vj914h88a4m90vy7/artifacts/x64%2FRelease%2FCChromaEditorLibrary.dll
32 bits https://ci.appveyor.com/api/buildjobs/km9p8klg1xshva8v/artifacts/Release%2FCChromaEditorLibrary.dll

Thanks for your work.

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+os:
+  - Visual Studio 2017
+
+# Win32 and x64 are CMake-compatible solution platform names.
+# This allows us to pass %PLATFORM% to CMake -A.
```
Comment (mercuriete) [inline .appveyor.yml]: This comment is useless... I will delete it.

**Comment by Riesi** [comment]
Does that even make sense when I don't even update the tree to follow the official source? (I don't plan to update ever)

I just deleted a bunch of code and replaced returns in a way that I guessed makes sense.

The best solution would be to create a proper stub-dll without shipping all of the CChromaEditor stuff.

**Comment by mercuriete** [comment]
ok, you can close this PR if you want.
I don't have a windows pc to compile the project so this is why I needed to use a continous integration system.

Your project is awesome BTW.

**Comment by Riesi** [comment]
I don't compile directly on Windows either. I spin-up a VM when I have to touch it.

fwiw I don't plan to rebase the code anytime soon, so CI doesn't really make sense.
But I guess you have CI working on your repo, so you should be fine.

Have a nice day.

**Pull request closed without merging**